src/adr/0034-mailbox-api-visibility-and-usage.md

```diff
@@ -0,0 +1,39 @@
+# 34. Mailbox API visibility and usage
+
+Date: 2020-04-27
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+For some reasons, all mailboxes implementations rely on some abstract classes located in `mailbox-store` module (like 
```

Comment:
Can you rephrase the sentence to not start it by `For some (unexplained) reasons,`

From a reviewer perspective, it avoids asking the question (Why?). Removing the prefix is ok to me.